src/node/history.h

```diff
@@ -7,6 +7,7 @@
 #include "ds/dl_list.h"
 #include "ds/logger.h"
 #include "ds/thread_messaging.h"
+#include "endian.h"
```

Comment:
Unused?

Reply:
Surprisingly used by the base64 code, will check if that's still the case.

Reply:
Nope, but used by merklecpp:

```
In file included from ../src/node/history.h:25:
../src/libmerklecpp/./merklecpp.h:64:22: error: use of undeclared identifier 'htobe64'
    size_t size_be = htobe64(size);
                     ^
../src/libmerklecpp/./merklecpp.h:77:12: error: use of undeclared identifier 'be64toh'
    return be64toh(result);
           ^
../src/libmerklecpp/./merklecpp.h:1529:16: error: use of undeclared identifier 'be32toh'
      cws[i] = be32toh(((int32_t *)block)[i]);
               ^
../src/libmerklecpp/./merklecpp.h:1561:35: error: use of undeclared identifier 'htobe32'
      ((uint32_t*)out.bytes)[i] = htobe32(s[i] + h[i]);
                                  ^
../src/libmerklecpp/./merklecpp.h:1580:35: error: use of undeclared identifier 'htobe32'
      ((uint32_t*)out.bytes)[i] = htobe32(((uint32_t*)ctx.h)[i]);
                                  ^
../src/libmerklecpp/./merklecpp.h:1611:35: error: use of undeclared identifier 'htobe32'
      ((uint32_t*)out.bytes)[i] = htobe32(ctx.state[i]);
                                  ^
6 errors generated.
```

Reply:
Interesting! merklecpp uses those legitimately, but I didn't need that #include so far. I'll look for a more portable solution tomorrow.